improver/calibration/ensemble_calibration.py

```diff
@@ -122,6 +126,10 @@ def __init__(self, tolerance=0.02, max_iterations=1000):
                 predictor_of_mean is "realizations", then the number of
                 iterations may require increasing, as there will be
                 more coefficients to solve for.
+            each_point (bool):
+                If True, coefficients are calculated independently for each
+                point within the input cube by minimising each point
+                independently.
```

Comment:
I've renamed the variable for determining whether coefficients will be calculated independently within this class to `point_by_point`, as suggested, to avoid any confusion with the `each_point` and `minimise_each_point` variables used in the `EstimateCoefficientsForEnsembleCalibration` class.